src/sentry/api/bases/dashboard.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import absolute_import
+from rest_framework import serializers
+from sentry.models import WidgetDisplayTypes
+
+
+class DashboardSerializer(serializers.Serializer):
+    title = serializers.CharField(required=True)
+    # TODO(lb): Can't figure out how to represent a json blob.
+    # data = ListField(
+    #     child=serializers.CharField(),
+    #     required=False,
+    #     allow_null=True,
+    # )
+
+
+class WidgetSerializer(serializers.Serializer):
```

Comment:
Hmm... this probably shouldn't be in this PR. I can add it to https://github.com/getsentry/sentry/pull/11398